app/src/main/java/org/p2p/wallet/rpc/RpcConstants.kt

```diff
@@ -0,0 +1,10 @@
+package org.p2p.wallet.rpc
+
+object RpcConstants {
+
+    const val REQUEST_METHOD_KEY = "method"
+    const val GET_CONFIRMED_TRANSACTIONS = "getConfirmedTransaction"
+
+    const val REQUEST_PARAMETER_KEY = "encoding"
```

Comment:
there are a lot of request parameters, so i suggest you add that this is encoding parameter
`REQUEST_PARAMETER_KEY_ENCODING`
and
`REQUEST_PARAMETER_VALUE_JSON_PARSED`